docs/topics/proc_deploying-a-booster-to-openshiftcontainerplatform.adoc

```diff
@@ -1,15 +1,16 @@
 [id='deploying-a-booster-to-openshiftcontainerplatform_{context}']
-= Deploying a booster to {OpenShiftContainerPlatform}
+[id='deploying-an-example-to-openshiftcontainterplatform_{context}']
+= Deploying an example to {OpenShiftContainerPlatform}
 
 .Prerequisites
 
-* Have a booster created using link:{link-launcher-oso}[{name-launcher-oso}] or the {name-launcher} tool in xref:creating-and-deploying-a-booster-using-your-openshiftlocal_{context}[a {OpenShiftLocal}].
+* Have an example created using link:{link-launcher-oso}[{name-launcher-oso}] or the {name-launcher} tool in xref:creating-and-deploying-a-booster-using-your-openshiftlocal_{context}[a {OpenShiftLocal}].
 * Have access to an {OpenShiftContainerPlatform} Web console.
 * Have the `oc` CLI client installed.
 
 .Procedure
 
-. Clone your booster Github repository, or download and extract the booster ZIP archive.
+. Clone your example Github repository, or download and extract the example ZIP archive.
```

Comment:
This sentence IMO makes it sound there's a generic example repository on GitHub, as opposed to a specific repository with an example application.

Reply:
Agree. I updated the wording to make it more clear.